internal/goutils/pointers.go

```diff
@@ -1,4 +1,4 @@
-package handy
+package goutils
```

Comment:
Can you remove the `Str`, `Bool` and `Int64` functions? They don't really save any lines of code and makes the code that uses these functions more confusing the read since someone has to figure out what these functions do. Simply using the `&` operator is more intuitive.

Reply:
In golang you cannot do this. Address operator cannot be applied to literals: (reference: https://go.dev/ref/spec#Address_operators)
Ex: 
```var textPtr *string =  &"some text" // illegal```

Therefore, there will be 2 lines to express this. First create a variable and then take address. This is the reason I named it `goutils`. It is purely related to golang "issues".

Alternativly, there is a library: https://pkg.go.dev/github.com/icza/gog#Ptr but adding dependency for this case is an overkill in my opinion.

Just by reading more about this it seems that we can replace all functions with one generic. These methods were created just a long time ago before I started using generics in connectors lib.